src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/IverksettingRepository.kt

```diff
@@ -33,19 +36,16 @@ class IverksettingRepository(val namedParameterJdbcTemplate: NamedParameterJdbcT
         namedParameterJdbcTemplate.update(sql, mapSqlParameterSource)
     }
 
-
-    private fun lagreIverksett(behandlingId: UUID, iverksett: Iverksett, brev: Brev) {
+    private fun lagreIverksett(behandlingId: UUID, iverksett: Iverksett) {
         val sql = "INSERT INTO iverksett VALUES(:behandlingId, :iverksettJson::JSON)"
         val iverksettString = objectMapper.writeValueAsString(iverksett)
-
         val mapSqlParameterSource = MapSqlParameterSource(
-                mapOf(
+                mapOf<String?, Any>(
```

Comment:
Er ikke denne unødvendig? `MapSqlParameterSource` tar kun in `mapOf<String?, Any>` mapOf automatiskt blir den typen?